packages/app/src/server/service/search.ts

```diff
@@ -73,8 +75,18 @@ class SearchService implements SearchQueryParser, SearchResolver {
       logger.error(err);
     }
 
+    this.isElasticsearchReindexOnBoot = this.configManager.getConfig('crowi', 'app:elasticsearchReindexOnBoot');
+
     if (this.isConfigured) {
-      this.fullTextSearchDelegator.init();
+      if (!this.isElasticsearchReindexOnBoot) {
```

Comment:
I think you just need to get this `if (!this.isElasticsearchReindexOnBoot)` block out of the `if(this.isConfigured)` block and put it after the `if (!this.isElasticsearchReindexOnBoot)` 

so the deleted code `this.fullTextSearchDelegator.init();` should get back to its original position

Reply:
Done

Reply:
I brought back `if (!this.isElasticsearchReindexOnBoot) {` inside `if(this.isConfigured)` conditions, since before update made error on `Node CI for app development` - yarn test session

Reply:
I'm sorry maybe my direction was wrong. 
I think codes should be put in the `if(this.isConfigured)` so you don't need to take it out.
I will have yuki check the PR again so pls work on the other comment.